package.json

```diff
@@ -25,13 +25,17 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "babel src/ -d lib/",
+    "deploy": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "flow": "flow",
-    "devserver": "NODE_ENV=development nodemon ./lib/backend/server.js",
-    "prodserver": "NODE_ENV=production node ./lib/backend/server.js"
+    "devserver": "NODE_ENV=development nodemon ./lib/backend/server.js"
+  },
+  "proxy": {
+    "/api": {
+      "target": "http://localhost:3001"
+    }
   },
-  "proxy": "http://localhost:3001",
```

Comment:
Updated this since because of the slight updates in the API route URLs. This is more for development, so when we have both the devserver and the front-end running locally on our machines, the api calls will be proxied correctly.

Reply:
Is there any way to test this without committing? Just want to make sure it'll still allow the api to work correctly.

Reply:
Just test the branch locally by running both the Express server with `npm run devserver` and and the frontend with `npm start` (you'll need multiple terminals to do this, for now). Then see if when you do api calls from the frontend, you get the responses from the api. @Zexdur